tests/accessor_tests.cpp

```diff
@@ -0,0 +1,163 @@
+#include <cstdlib>
+#include <random>
+
+#include <catch2/catch_approx.hpp>
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/benchmark/catch_benchmark.hpp>
+
+#include <glm/glm.hpp>
+#include <glm/gtc/type_ptr.hpp>
+#include <glm/gtx/quaternion.hpp>
+#include <glm/gtx/matrix_decompose.hpp>
+
+#include "base64_decode.hpp"
+#include "fastgltf_parser.hpp"
+#include "fastgltf_types.hpp"
+#include "fastgltf_tools.hpp"
+#include "gltf_path.hpp"
+
+template<class... Ts> struct overloaded : Ts... { using Ts::operator()...; };
+template<class... Ts> overloaded(Ts...) -> overloaded<Ts...>;
```

Comment:
As they're now doubled in gl_viewer and here, I think these should be moved into fastgltf_util.hpp as a convenience tool for other developers.

Reply:
Seems like a good idea, although they would probably need a more descriptive name. In my own engine I call these "VisitorBase" since their job is to serve as a visitor base class.

Reply:
Or just simply "visitor". It would ultimately be fastgltf::visitor which I think would be perfectly fine.

Reply:
Added fastgltf::visitor in [9a35575](https://github.com/spnda/fastgltf/pull/20/commits/9a35575e79810ed5ef622f8f487d83f5f9fab50b).